requirements.txt

```diff
@@ -1,3 +1,4 @@
-Padding>=0.4
```

Comment:
I don't think we should update requirements in this PR. Instead of adding `responses` here, let's add it to test requirements in `setup.py` like `tests_require = ['nose', 'mock', `responses`]`